docs/release-notes/8.17.asciidoc

```diff
@@ -1,6 +1,60 @@
 [[release-notes-header-8.17.0]]
 == 8.17
 
+[discrete]
+[[release-notes-8.17.5]]
+=== 8.17.5
+
+[discrete]
+[[known-issue-8.17.5]]
+==== Known issues
+
+// tag::known-issue[]
+[discrete]
+.Duplicate alerts can be produced from manually running threshold rules 
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running threshold rules could produce duplicate alerts if the date range was already covered by a scheduled rule execution.
+====
+// end::known-issue[]
+
+// tag::known-issue[]
+[discrete]
+.Manually running custom query rules with suppression could suppress more alerts than expected
+[%collapsible]
+====
+*Details* +
+On November 12, 2024, it was discovered that manually running a custom query rule with suppression could incorrectly inflate the number of suppressed alerts. 
+====
+// end::known-issue[]
+
+// tag::known-issue[]
+[discrete]
+.Installing an {elastic-defend} integration or a new agent policy upgrades installed prebuilt rules, overwriting user-added actions and exceptions
+[%collapsible]
+====
+*Details* +
+When you install an {elastic-defend} integration or a new agent policy for this integration, all the installed prebuilt detection rules are upgraded to their latest versions (if any new versions are available). The upgraded rules lose any user-added rule actions and exceptions.
```

Comment:
Missing the discover date. Also, should we order these reverse chron order so the newest ones appear at the top? 
```suggestion
On April 10, 2025, it was discovered that when you install an {elastic-defend} integration or a new agent policy for this integration, all the installed prebuilt detection rules are upgraded to their latest versions (if any new versions are available). The upgraded rules lose any user-added rule actions and exceptions.
```

Reply:
I'll add the date in a separate PR since this change would affect additional release note files outside of the 8.17.0 one. As for the other known issues, I'll likely remove them in the near future and fold them into the docs. [Here's](https://elastic.slack.com/archives/C071E7FCR1A/p1742908992998569?thread_ts=1742821691.182699&cid=C071E7FCR1A) the Slack thread on that.